app/templates/components/public/featured-speaker-list.hbs

```diff
@@ -0,0 +1,8 @@
+<h3>{{t 'Featured Speakers'}}</h3>
+<div class="speaker-list ui grid">
+  {{#each speakers as |speaker|}}
+    {{#if speaker.featured}}
```

Comment:
Instead of looping through the entire list and checking, provide a filtered computed property called as `featuredSpeakers` and loop through that instead.